internal/runners/secrets/secrets.go

```diff
@@ -185,6 +117,28 @@ func filterSecrets(secrectDefs []*secretsModels.SecretDefinition, filter string)
 	return secrectDefsFiltered
 }
 
+type secretExports []*SecretExport
+
+func (es secretExports) MarshalOutput(format output.Format) interface{} {
+	switch format {
+	case output.JSONFormatName, output.EditorV0FormatName, output.EditorFormatName:
+		return es
+
+	default:
+		rows, fail := secretsToRows(es)
+		if fail != nil {
+			return fail.WithDescription(locale.T("secrets_err_output"))
+		}
+
+		t := gotabulate.Create(rows)
+		t.SetHeaders([]string{locale.T("secrets_header_name"), locale.T("secrets_header_scope"), locale.T("secrets_header_value"), locale.T("secrets_header_description"), locale.T("secrets_header_usage")})
+		t.SetHideLines([]string{"betweenLine", "top", "aboveTitle", "LineTop", "LineBottom", "bottomLine"}) // Don't print whitespace lines
+		t.SetAlign("left")
+
+		return t.Render("simple")
```

Comment:
Pretty sure you can just drop this to get the table support

Reply:
The structures for plain and json outputs are different. I've started a branch with some work on that, and will link it in the relevant story.